src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/arraytrailingcomma/InputArrayTrailingComma.java

```diff
@@ -58,4 +58,20 @@
             1,
             2
             ,};
+
+    Object[][] g1 = new Object[][]
+    {
+        { 1, 1 },
+        {
+           null,
+           new int[] { 2,
+                   3 }, }, };
+
+    Object[][] g2 = new Object[][]
+    {
+        { 1, 1 },
+        {
+           null,
+           new int[] { 2,
+                   3 } } };
```

Comment:
Please add `// violation` comments on lines with violation (for all violations in file)